srcpkgs/wluma/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'wluma'
+pkgname=wluma
+version=4.9.0
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config clang cargo"
+makedepends="eudev-libudev-devel v4l-utils-devel wayland-devel vulkan-loader-devel
+ dbus-devel"
+short_desc="Automatic brightness adjustment based on screen contents and ALS"
+maintainer="Emil Miler <em@0x45.cz>"
+license="ISC"
+homepage="https://github.com/maximbaz/wluma"
+changelog="https://github.com/maximbaz/wluma/releases"
+distfiles="https://github.com/maximbaz/wluma/archive/refs/tags/${version}.tar.gz"
+checksum=b7ff8decf697975b85ea37d4d6f3affc0c06b69553acca611cf2efedc92825b8
+
+post_install() {
+	vinstall 90-wluma-backlight.rules 0644 /usr/lib/udev/rules.d/
+}
+
+post_install() {
+	vlicense LICENSE
```

Comment:
You have duplicate `post_install` functions, the first one will be skipped afaik.

Reply:
How did i miss this?! 😅